improve github copilot (#926)
Worklytics/psoxy: docs/sources/github/copilot/README.md

- If `github_copilot_installation_id` is not set, authentication URL will not be properly formatted and you will see *401: Unauthorized* when trying to get an access token.
- If you see *404: Not found* in logs please review the *IP restriction policies* that your organization might have; that could cause connections from psoxy AWS Lambda/GCP Cloud Functions be rejected.

6. Update the configuration parameters in your host platform with values obtained in previous step:

>     - `PSOXY_GITHUB_COPILOT_CLIENT_ID` with `App ID` value. **NOTE**: It should be `App Id` value as we are going to use authentication through the App and **not** *client_id*.

Reviewer: ```suggestion
    - `PSOXY_GITHUB_COPILOT_CLIENT_ID` with `Client ID` value from the application you have created.
```
